### DISCUSSION TITLE
質問に対しての「コメントする」「回答する」フォームの改善要望

### DISCUSSION THREAD

**Discussion by mariori12** [increments/qiita-discussions#494] [要望・アイデア・不具合]
お世話になっております。

表題の件について、
「コメントに対し返信する場合にMarkdownが適用されない」という
元々の仕様への対処かと存じますが対処後のUIについて、
「コメントする」、「回答する」フォームが同様かつ位置が近いこともあり
「新規に回答しようとしたら既回答のコメントとして記述してしまった」
といった事象が時折発生しております。(自身・他者の回答含め)

可能でしたら、ひと目で分かるよう、
フォームの位置・見た目等に違いを設けていただければ幸いです。

宜しくお願いいたします。

**Comment by kyntk** [comment]
@mariori12 ご意見ありがとうございます！
確かに分かりづらくなっていると思われるので、回答とコメントの違いが分かりやすくなるように改善いたします :bow:

**Reply by mariori12** [reply]
@kyntk 
お世話になっております。

本コメント記載時点で表題の件につきまして
より顕著に事象が見受けられるようになりましたが、
改善の目処はございますでしょうか。

確認の程宜しくお願いいたします。

**Reply by tanaka-asami415** [reply]
@mariori12 
お待たせして申し訳ありません。
8月中に着手予定ですので、もうしばらくお待ちいただけると幸いです🙇

**Comment by kimkim0814** [comment]
@mariori12
お待たせいたしました:bow:

質問に対しての「コメント」と「回答」のフォームについて、違いが分かりやすくなるよう変更いたしました。

「コメント」に関してはGithubのリプライの仕様のように、クリックしたら
フォームが広がるようになっております。

是非お試しください:blush:


![回答コメント](https://github.com/increments/qiita-discussions/assets/54876760/6e420cdb-4a9f-4170-8725-cf7c5967dd81)
